src/test/resources/python/regex/requirementsAllKindOfDepsVariations.txt

```diff
@@ -0,0 +1,7 @@
+SomeProject1
```

Comment:
Let's add some cases for requirements without spaces, for example:
SomeProject8 ==6.7
SomeProject9== 9.8
SomeProject10==9.0